src/fixate/core/exceptions.py

```diff
@@ -1,32 +1,32 @@
-class SequenceAbort(BaseException):
+class SequenceAbort(Exception):
```

Comment:
Since these are all getting touched, It's probably worth making a top level `FixateError` and deriving all the more specific errors from that.